### PR TITLE
Pimcore 3 Compatibility

### PR DESCRIPTION
**Pull request by gsomoza**
A few changes for better Pimcore 3 compatibility:
- Switched to long PHP open tags (<?php).
- Started using Namespaces.
- Cleaned up views (minor).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+Provides Bootstrap-specific areas for your themes.
+
+Installation
+------------
+
+Using Composer:  
+
+```
+composer require pfaffenbauer/bootstrap
```
Comment (Hagith) [inline README.md]: Package was moved under pimcore-extensions community roof. Please change to "pimcore-extensions/bootstrap"

File: composer.json [hunk 2]
```diff
@@ -1,10 +1,10 @@
 {
-    "name": "pimcore-extensions/bootstrap",
+    "name": "pfaffenbauer/bootstrap",
```
Comment (Hagith) [inline composer.json]: As above, please revert to "pimcore-extensions/bootstrap"

File: composer.json [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
     "type": "pimcore-plugin",
     "license": "Simple BSD",
     "description": "Adds Bootstrap JS and CSS Files and some Area-Bricks",
     "keywords": ["pimcore","bootstrap"],
-    "homepage": "https://github.com/pimcore-extensions/bootstrap",
+    "homepage": "https://github.com/dpfaffenbauer/pimcore-bootstrap",
```
Comment (Hagith) [inline composer.json]: Same here :)

File: controllers/BootstrapController.php [hunk 4]
```diff
@@ -32,4 +36,9 @@ public function alertAction()
     {
 
     }
+
+    public function __construct(Zend_Controller_Request_Abstract $request, Zend_Controller_Response_Abstract $response, array $invokeArgs = array())
```
Comment (Hagith) [inline controllers/BootstrapController.php]: It's not needed.

File: README.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+```
+
+Composer will automatically copy the area files into your `website/var/areas` directory.
```
Comment (Hagith) [inline README.md]: Views are copied by installation from admin panel, not by composer itself.

File: README.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+Next, you'll have to make the views available. You can either copy the folders inside `views/scripts` into 
+`website/views/scripts` or otherwise:
```
Comment (Hagith) [inline README.md:18]: I'm not sure it's required step, it was working without this. I will double check on latest pimcore version.

File: plugin.xml [hunk 7]
```diff
@@ -1,43 +1,43 @@
...
-
-			<!-- css files with path relative to plugin-directory -->
-			<pluginCssPaths>
-                <path>/plugins/Bootstrap/static/css/admin.css</path>
-			</pluginCssPaths>
-		</plugin>
-</zend-config>
+<?xml version="1.0"?>
+<zend-config xmlns:zf="http://framework.zend.com/xml/zend-config-xml/1.0/">
+		<plugin>
```
Comment (Hagith) [inline plugin.xml]: Thanks for changing to spaces, it wold be great if you fix indentation too.

File: README.md (hunk 6 above)
Comment (gsomoza) [inline README.md:18]: It was necessary for me with a relatively clean installation of the most recent release + sample data. But if you know a better way that would be awesome.

**Comment by Hagith** [comment]
LGTM. @dpfaffenbauer?

PS: I will check view scripts path as soon I have some free time. For now it's OK. If there is better option will present it as another PR.

**Comment by Hagith** [comment]
As this PR becomes conflicted I prepared manual rebase here #10
Please review/test it if you can. Thank you for contribution.

PS: I will test this version in project which I'm implementing right now.

**Comment by gsomoza** [comment]
Glad I could help!

**Pull request closed without merging**